### PR TITLE
feat(dedicated): eol rbx1 banner

### PR DESCRIPTION
**Pull request by qpavy**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #MANAGER-12907
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~~[] Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by antonymarion — DISMISSED** [review]

**Review by antonymarion — APPROVED** [review]

File: packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/servers.html [hunk 1]
```diff
@@ -3,6 +3,11 @@
         data-heading="{{ :: 'dedicated_servers_title' | translate }}"
     ></oui-header>
 
+    <!-- RBX1 EOL Banner-->
+    <ovh-manager-rbx-eol-banner
+        data-show="$ctrl.displayRbx1EolBanner.rbx1Eol"
```
Comment (ghyenne) [inline packages/manager/apps/dedicated/client/app/dedicated/dedicated-server/servers/servers.html:8]:
```suggestion
        data-ng-if="$ctrl.displayRbx1EolBanner.rbx1Eol"
```

File: packages/manager/apps/dedicated/client/app/app.module.js [hunk 2]
```diff
@@ -90,6 +90,8 @@ import chartjs from 'angular-chart.js';
 
 import moduleExchange from '@ovh-ux/manager-exchange';
 import ovhManagerFilters from '@ovh-ux/manager-filters';
+// Import Hub components for rbx1 eol banner
+import ovhManagerHub from '@ovh-ux/manager-hub';
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/client/app/app.module.js:94]: **nitpick(blocking):** We cannot import Hub into dedicated. Please find a more suitable place to define the component.

**Review by anooparveti — CHANGES_REQUESTED** [review]

File: packages/manager/apps/hub/src/dashboard/dashboard.html [hunk 3]
```diff
@@ -100,6 +100,10 @@
                 ></span>
             </a>
         </oui-message>
+        <!-- RBX1 EOL Banner-->
+        <ovh-manager-rbx-eol-banner
+            data-show="$ctrl.displayRbx1EolBanner.rbx1Eol"
```
Comment (ghyenne) [inline packages/manager/apps/hub/src/dashboard/dashboard.html:105]:
```suggestion
            data-ng-if="$ctrl.displayRbx1EolBanner.rbx1Eol"
```

File: packages/manager/apps/dedicated/package.json [hunk 4]
```diff
@@ -38,6 +38,7 @@
     "@ovh-ux/manager-error-page": "^2.3.6",
     "@ovh-ux/manager-exchange": "^4.1.2",
     "@ovh-ux/manager-filters": "^1.1.1",
+    "@ovh-ux/manager-hub": "^3.7.14",
```
Comment (anooparveti) [inline packages/manager/apps/dedicated/package.json:41]: **nitpick:** You dont need this anymore

File: packages/manager/modules/cloud-universe-components/src/rbx-eol-banner/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "rbx1_eol_banner": "ATTENTION: Pour permettre la modernisation du bâtiment, les serveurs situés au Datacenter Roubaix RBX1 doivent être migrés avant le 31 décembre 2024. Merci de vous référer au mail reçu si vous avez des serveurs concernés. <a href=\"{{link}}\" target=\"_blank\">Consultez notre guide de migration.</a>"
```
Comment (anooparveti) [inline packages/manager/modules/cloud-universe-components/src/rbx-eol-banner/translations/Messages_fr_FR.json:2]: **nitpick:** Link can be separated into a different key and define the anchor tag in template.

**Review by anooparveti — CHANGES_REQUESTED** [review]

**Review by anooparveti — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by ghyenne — DISMISSED** [review]

**Review by dectotam — APPROVED** [review]

**Review by JacquesLarique — DISMISSED** [review]

**Review by antonymarion — APPROVED** [review]

**Review by JacquesLarique — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**